cypress/e2e/course_board/createEditDeleteColumnInBoard.feature

```diff
@@ -0,0 +1,35 @@
+@release
+Feature: To create, edit and delete column in the course board
+
+As a teacher I want to ceate, edit and delete the column in the course board so that I can manage the colum in the board.
```

Comment:
typo 

- ceate
- colum

Reply:
no its in this sentence

`I can manage the colum in the board`

Reply:
done!